tests/h/services/group_test.py

```diff
@@ -197,45 +199,5 @@ def test_wraps_user_service_as_user_fetcher(self, pyramid_request, user_service)
 
 
 @pytest.fixture
-def usr_svc(db_session):
-    def fetch(userid):
-        # One doesn't want to couple to the user fetching service but
-        # we do want to be able to fetch user models for internal
-        # module behavior tests
-        return db_session.query(User).filter_by(userid=userid).one_or_none()
-
-    return fetch
-
-
-@pytest.fixture
-def svc(db_session, usr_svc):
-    return GroupService(db_session, usr_svc)
-
-
-class GroupWithName(Matcher):
-    """Matches any Group with the given name."""
-
-    def __init__(
-        self, name
-    ):  # pylint:disable=super-init-not-called #:Overwriting __eq__ instead
-        self.name = name
-
-    def __eq__(self, group):
-        """Return True if group is instance of :class:h.models.Group and has matching name."""
-        if not isinstance(group, Group):
-            return False
-        return group.name == self.name
-
-
-class GroupScopeWithOrigin(Matcher):
-    """Matches any GroupScope with the given origin."""
-
-    def __init__(
-        self, origin
-    ):  # pylint:disable=super-init-not-called #: Overwriting __eq__ instead
-        self.origin = origin
-
-    def __eq__(self, group_scope):
-        if not isinstance(group_scope, GroupScope):
-            return False
-        return group_scope.origin == self.origin
```

Comment:
A lot of different kinds of not important going on in here:

* The user service sort of creates a real one, but then doesn't rely on it at all. We could actually use the real one or a service mock, and the mock works and is less effort so I chose that one
* The `GroupWithName` matcher has been replaced with `h-matchers`
* The `GroupScopeWithOrigin` just isn't used at all?